### PR TITLE
Fix: Set fee_rate

### PR DESCRIPTION
**Pull request by rahulghangas**
Fixes error of fallback fee while simulating mining

### CODE REVIEW

File: infra/bitcoin/run.sh [hunk 1]
```diff
@@ -9,9 +9,10 @@ sleep 20
 # Print setup
 echo "BITCOIN_ADDRESS=$ADDRESS"
 
-# Create wallet
 /app/bin/bitcoin-cli createwallet "testwallet"
 
+sleep 10
```
Comment (jazg) [inline infra/bitcoin/run.sh:14]: Is the sleep necessary?

File: infra/bitcoin/run.sh [hunk 2]
```diff
@@ -28,6 +29,6 @@ do
     /app/bin/bitcoin-cli -regtest generatetoaddress 1 $ADDRESS
     sleep 5
     # send tx to own address while paying fee to the miner
-    /app/bin/bitcoin-cli -regtest sendtoaddress $ADDRESS 1 "" "" true
+    /app/bin/bitcoin-cli -regtest -named sendtoaddress address=$ADDRESS amount=0.1 subtractfeefromamount=false fee_rate=1
```
Comment (jazg) [inline infra/bitcoin/run.sh:32]: Did the amount we're sending ourselves get reduced from 1 to 0.1?

File: infra/bitcoin/run.sh (hunk 1 above)
Comment (rahulghangas) [reply infra/bitcoin/run.sh:14]: No it's not, that just crept in from debugging

File: infra/bitcoin/run.sh (hunk 2 above)
Comment (rahulghangas) [reply infra/bitcoin/run.sh:32]: I was just checking different amounts. It doesn't matter though, this transaction is there to only simulate mining, so the amount is irrelevant as long as it covers fees

File: infra/bitcoin/run.sh (hunk 1 above)
Comment (rahulghangas) [inline infra/bitcoin/run.sh:14]:
```suggestion
```

File: infra/bitcoin/run.sh (hunk 2 above)
Comment (jazg) [reply infra/bitcoin/run.sh:31]: Doesn't that get sent to the primary account we use for testing?

Comment (rahulghangas) [reply infra/bitcoin/run.sh:31]: I don't follow, `$ADDRESS` is the main account that we are using. Then we simulate mining wherein we generate blocks to `$ADDRESS` and send some amount `x` from `$ADDRESS` to itself.

Comment (rahulghangas) [inline infra/bitcoin/run.sh:31]:
```suggestion
    /app/bin/bitcoin-cli -regtest -named sendtoaddress address=$ADDRESS amount=0.1 subtractfeefromamount=true fee_rate=1
```
This needs to be true

Comment (jazg) [reply infra/bitcoin/run.sh]: Ah ok, got it.

**Review by jazg — APPROVED** [review]

**Review by roynalnaruto — APPROVED** [review]
This one looks good. Tested building-running the image 👍

**Pull request merged**